packages/office-ui-fabric-react/src/components/Nav/Nav.scss

```diff
@@ -96,6 +96,11 @@ $linkRightPadding: 20px;
   vertical-align: middle;
 }
 
+.ms-Nav-linkHorizontal {
+  @include float(left);
+  display: inline;
```

Comment:
Just curious, if `float:left / float:right` is the only requirement, Can't we just provide `className` on `<ul>` and `<li>`s and let consumer take care of overriding this style whichever way they want. Do we really need an additional option for one style? 
👍 for adding an example that demonstrate horizontal styling.